plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/style.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-woocommerce-add-to-cart-with-options {
+	.variations_form.cart {
```

Comment:
Not a big deal, but highlighting that the markup using this class is introduced in the other PR (https://github.com/woocommerce/woocommerce/pull/53947/) if I'm not wrong.

Reply:
Good call! I removed it from this PR and will add it to https://github.com/woocommerce/woocommerce/pull/53947.